packages/fuselage/src/components/Message/mixins.scss

```diff
@@ -27,6 +28,14 @@
       margin-block: lengths.margin(-2);
     }
   }
+
+  & .rcx-attachment {
+    &__details {
+      .rcx-message-body {
+        color: colors.neutral(700);
```

Comment:
The `.rcx-attachment__details` is an app component. I don't think we should have app style here on fuselage. I guess the right place for this [style is here](https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/client/components/message/Attachments/QuoteAttachment.tsx#L17).

Reply:
Yep, you're right, thanks @filipemarins .... closing this PR to include this change at [Core Pull Request](https://github.com/RocketChat/Rocket.Chat/pull/26751)